src/app/home/home-gallery/home-gallery.component.html

```diff
@@ -0,0 +1,27 @@
+
+<div style="padding-left: 10%; padding-right:10%">
+  <div class="container">
+    <div class="row" >
+
+        <div class="col-lg-4 col-md-6 col-xs-12 " *ngFor="let links of pictureArray">
+            <img class="gallery_product" src="{{links}}" >
+            <br>
+             <span class="res-name-style">Restaurant Name</span>
+             <br>
+            <span><app-rating></app-rating>    (1745) $$$$</span> 
+            <hr>
+            <span class="res-type-style">Italian | International | Mediteran  </span>
+            <br>
+            <br>
+            <button class="button button_reserve">Reserve now</button>
```

Comment:
`button-reserve` would be better than `button_reserve`. Generally use `-` instead of `_` in css class names